tests/aruba/aruba.md

```diff
@@ -0,0 +1,50 @@
+# Aruba-rspec testframework for CB CLI
+The aruba test project is located in tests/aruba/
+
+## E2E tests
+
+Run without Docker:
+```
+curl -sSL https://get.rvm.io | bash -s stable --ruby
+source /Users/mhalmy/.rvm/scripts/rvm
```

Comment:
* Would be useful here to refer to the [Official RVM Installation instructions](https://rvm.io/rvm/install) as well.
* This path is not valid for everyone. Please make it general with; as a last step from the console and based on the Official RVM Installation instructions:
> The rvm function will be automatically configured for every user on the system if you install as single user. Read the output of installer to check which files were modified.
 ```* To start using RVM you need to run `source /Users/<your user>/.rvm/scripts/rvm`
    in all your open shell windows, in rare cases you need to reopen all shell windows.```
For `Multi-User` please follow [Official RVM Installation instructions](https://rvm.io/rvm/install)...etc.